tools/topology/topology1/sof-acp-rmb-tdm8ch-dmic2ch-nau8825-max98360.m4

```diff
@@ -32,13 +32,17 @@ VIRTUAL_WIDGET(ACPHS1 OUT, output, 3)
 #
 # Define the pipelines
 #
-# PCM0 ----> buffer ------+
-#                         |----> muxdemux ----> buffer ---->  ACPSP_play
-# PCM1 ----> buffer ----->
 #
-# PCM0 <---- muxdemux <---- buffer <---- ACPSP_Capture
+#              |-------------->echoref --->ACPHS1
```

Comment:
im a little confused by this pictogram, it shows data coming from the host into the echoref? also im confused why its linked at the buffer and not a mux

Reply:
The echo-ref is a virtual component without any functionality (echo-ref will not read/write from/to buffer), it is used only for termination of the ACPHS1 (2nd playback-dai of the mux).